components/brave_rewards/resources/rewards_panel/components/grant_captcha_challenge.tsx

```diff
@@ -39,6 +39,16 @@ export function GrantCaptchaChallenge (props: Props) {
       x: (rect.width / 2) - (event.clientX - rect.left),
       y: (rect.height / 2) - (event.clientY - rect.top)
     })
+
+    // Work around Linux-specific upstream bug by explicitly setting the drag
+    // image offset when initiating the drag
+    // See https://bugs.chromium.org/p/chromium/issues/detail?id=1297990
+    if (window.navigator.userAgent.includes('Linux') && devicePixelRatio > 1) {
+      event.dataTransfer.setDragImage(
```

Comment:
Do you think this is this something that will break the captcha (for this subset of users) when the upstream bug is fixed?

Reply:
I think it's possible, unfortunately, depending on how https://bugs.chromium.org/p/chromium/issues/detail?id=1297990 is ultiumately addressed. I tried to limit the impact as much as possible with the platform and `devicePixelRatio` guards. I feel like any workaround we implement will have to be removed and uplifted once the upstream bug is fixed. Definitely open to other approaches though!